### ISSUE TITLE
overview visualization

### ISSUE THREAD

**Issue by govvin** [maproulette/maproulette2#318]
When viewing the overview map of a challenge, it would be nice to have a convenient way to differentiate completed tasks versus pending tasks by way of pin colors.

Right now, it's just a pin (in blue) which you have to click. If it says "view", it's been fixed. Otherwise the color simply indicates the location of a particular task.

**Comment by mvexel** [comment]
Agreed that would be nice. I don't currently have the time to implement it but if you can (or know someone who can) help I would appreciate it!
